.github/scripts/end2end/configure-e2e.sh

```diff
@@ -109,3 +109,10 @@ sleep 10
 
 kubectl wait --for condition=DeploymentFailure=false --timeout 25m -n ${NAMESPACE} zenko/${ZENKO_NAME}
 kubectl wait --for condition=DeploymentInProgress=false --timeout 25m -n ${NAMESPACE} zenko/${ZENKO_NAME}
+
+
+if [ $ENABLE_RING_TESTS = true ]; then
+  # wait for ingestion processor to start consuming from Kafka
```

Comment:
this should (eventually) get baked into backbeat?
--> so we should probably add a ticket (and we can put it in the EPIC "cleanup kafka management" https://scality.atlassian.net/browse/ARTESCA-9180)

Reply:
https://scality.atlassian.net/browse/BB-655